integration/src/binance/binance.ts

```diff
@@ -62,19 +61,18 @@ export function binanceTests(get: () => { wallet: HDWallet; info: HDWalletInfo }
       TIMEOUT
     );
 
-    test.skip(
+    test(
```

Comment:
Would rather keep the skip if we aren't actually asserting on anything so it doesn't look like the test is passing when it isn't doing anything (other than making sure it doesn't throw I guess)